src/ScoreCard.js

```diff
@@ -0,0 +1,41 @@
+function ScoreCard(frames) {
+
+  this.frames = [];
+
+  this.openFrames = frames;
+
+  this.getScore = function() {
+    var result = 0;
+    for (i=0;i<this.frames.length;i++) {
+      result += this.frames[i].score;
+    }
+    return result;
+  };
+
+  this.roll = function(pinsHit) {
```

Comment:
This function has too many statements. (12)<br>This function's cyclomatic complexity is too high. (5)